streams/src/main/scala/tofu/streams/Region.scala

```diff
@@ -0,0 +1,16 @@
+package tofu.streams
+
+import cats.effect.ExitCase
+
+trait Region[F[_], G[_], E] {
```

Comment:
How about parameterizing `ExitCase` as it's done in `tofu.Finally` so we can do something about zio-streams

Reply:
@Odomontois, is `Region[F[_], G[_], +Exit]` enough?

Reply:
To represent process exit you are using cats.effect.ExitCase structure, which is poorly abstracts zio.Exit or any other possible exit.
So I mean Region[F[_], G[_], Exit[_]] here

Reply:
@Odomontois, currently I swithed to `Region[F[_], G[_], +Exit]`, which can be `Region[F[_], G[_], ExitCase[Throwable]]` in case of CatsEffect, or `Region[F[_], G[_], zio.Exit[X, E]]`. Do we really need higher-kinded type parameter `Exit[_]` here?
https://github.com/TinkoffCreditSystems/tofu/blob/1864900066f9fa1d0663cfc7954ce8f6fb17130b/streams/src/main/scala/tofu/streams/Region.scala

Reply:
Sorry. My Bad. We can keep simple type here.